### PR TITLE
Support CentOS/RHEL as base OS for running tests

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

File: tests/tox.sh [hunk 1]
```diff
@@ -11,10 +11,21 @@ set -ex
 
 # setup
 #################################################################################
-# XXX this should probably not install system dependencies like this, since now
-# it means we are tied to an apt-get distro
-sudo apt-get install -y --force-yes docker.io
-sudo apt-get install -y --force-yes xfsprogs
+# Check distro and install deps
+if hash apt-get 2>/dev/null; then
```
Comment (leseb) [inline tests/tox.sh]: Do you using `command -v` instead?
Just for consistency across the project

Comment (djgalloway) [inline tests/tox.sh]: Makes sense.  Fixed.

**Comment by GregMeno** [comment]
jenkins lint

**Comment by guits** [comment]
jenkins test ceph_ansible-jewel-centos7_cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-jewel-xenial_cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-jewel-centos7_cluster

File: tests/tox.sh [hunk 2]
```diff
@@ -11,10 +11,21 @@ set -ex
...
+    sudo apt-get install -y --force-yes xfsprogs
+  else
+    sudo yum install -y docker
+    sudo yum install -y xfsprogs
+    if ! systemctl status docker >/dev/null; then
+      # daemon doesn't start automatically after being installed
+      sudo systemctl restart docker
+    fi
+    # Allow running `docker` without sudo
+    sudo chgrp $(whoami) /var/run/docker.sock
```
Comment (leseb) [inline tests/tox.sh]: Please double quote `"$(whoami)"`.

Comment (djgalloway) [inline tests/tox.sh]: fixed

**Pull request merged**